includes/Assets.php

```diff
@@ -420,6 +420,8 @@ public function enqueue_front_scripts() {
             'rounding_precision' => wc_get_rounding_precision(),
             'mon_decimal_point'  => wc_get_price_decimal_separator(),
             'product_types'      => apply_filters( 'dokan_product_types', [ 'simple' ] ),
+            'loading_img'        => plugins_url( 'assets/images/loading.gif', __FILE__ ),
```

Comment:
this code will reference dokan/include/ folder, hence will output invalid file, why don't you use DOKAN_PLUGIN_ASSEST instead ?